docs/references/nextjs/clerk-middleware.mdx

```diff
@@ -338,38 +338,45 @@ The `clerkMiddleware()` function accepts an optional object. The following optio
 
   ---
 
+  - `organizationSyncOptions?`
+  - <code>[OrganizationSyncOptions](#organization-sync-options) | undefined</code>
+
+  Used to activate a specific [organization](docs/organizations/overview) or [personal account](https://clerk.com/glossary#personal-account) based on URL path parameters. If there's a mismatch between the active organization in the session (e.g., as reported by [`auth()`](/docs/references/nextjs/auth)) and the organization indicated by the URL, the middleware will attempt to activate the organization specified in the URL.
```

Comment:
Done! 90ad16deafd71d832c22cce78e5d48619bb3179c

Because the link doesn't hop directly to a personal account definition, I added a little text highlight to the link, so it will look like this:

![Screenshot 2024-10-29 at 9 27 58 AM](https://github.com/user-attachments/assets/0e1dbc37-46c0-4cca-903c-a068e896dc7d)

But I can go with the vanilla link if you'd prefer!